README.rst

```diff
@@ -815,6 +815,8 @@ Changes
 - Fix CPU Busy metric on "CPU Utilization" graph. (ZPS-3531)
 - Fix 'no volume group' warning events during modeling. (ZPS-3475)
 - Add Idle, Interrupt, Soft Interrupt, Steal metrics on CPU Utilization graph. (ZPS-3547)
+- Enable better management of sevice events (ZPS-3616)
```

Comment:
Typo: "sevice" instead of "service".

The other bullet items end with period (.), but this one doesn't. They should be made consistent.